aiosmtpd/docs/handlers.rst

```diff
@@ -184,6 +184,19 @@ The following hooks are currently supported (in alphabetical order):
 
    Called during ``NOOP``.
 
+.. method:: handle_PROXY(server, session, envelope, proxy_data)
```

Comment:
Is this method signature correct? I had no success with a dedicated `proxy_data` header, but could grab the information from `session.proxy_data`.

Reply:
Oops, my bad. It _should_ be as the documentation indicated, but apparently I forgot to change the implementation.

I'll fix it in the next commitpush.